tests/PHPStan/Rules/Missing/data/missing-return.php

```diff
@@ -396,6 +396,12 @@ public function bodySpecifiedVoidTReturn3(): \Generator
 		return 2;
 	}
 
+	public function yieldInExpression(): \Generator
```

Comment:
The issue reported was detected by the MissingReturn rule. Not sure if there is a more suitable way to test (the test is that there is no error).